AppCenterPush/AppCenterPush/MSPush.m

```diff
@@ -389,4 +392,13 @@ - (BOOL)didReceiveRemoteNotification:(NSDictionary *)userInfo fromUserNotificati
   return NO;
 }
 
+- (void)authTokenContext:(__unused MSAuthTokenContext *)authTokenContext didReceiveAuthToken:(__unused NSString *)authToken {
+
+  // Make a copy of push token so that this code is thread safe.
```

Comment:
pointer copy, not value (we don't the copy flag on the property), I think the comment is confusing.
we could be more implicit like use a local reference to be thread safe?

Reply:
Fixed by adding `copy` attribute to `pushToken` property.

Reply:
Ok I was thinking of just fixing the comment instead actually, not sure you need a value copy here.

Reply:
Synced offline, should be copy for best practice.